dotCMS/src/main/java/com/dotcms/util/ConversionUtils.java

```diff
@@ -137,6 +137,38 @@ public static long toLong (final Object input, final Long defaultLong) {
 		return resultLong;
 	}
 
+	/**
+	 * Converts 1kb to 1024
+	 * Converts 1mb to 1024 * 1024
+	 * Converts 1gb to 1024 * 1024 * 1024
+	 * @param humanDisplaySize
+	 * @param defaultLong
+	 * @return long
+	 */
+	public static long toLongFromByteCountHumanDisplaySize (final String humanDisplaySize, final Long defaultLong) {
+
+		if (humanDisplaySize.length() > 2) {
+
+			final String postfix     = humanDisplaySize.substring(humanDisplaySize.length()-2);
+			final String stringValue = humanDisplaySize.substring(0, humanDisplaySize.length()-2);
+			long  value              = toLong(stringValue, defaultLong);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'value' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4772115)